pkg/controller/webhook/webhook.go

```diff
@@ -89,6 +108,41 @@ func notify(destination string, message []byte) error {
 		destination, resp.Status)
 }
 
+func notifyWS(destination string, message []byte) error {
+	ctx, cancel := context.WithTimeout(context.Background(), notificationSendTimeout)
+	defer cancel()
+
+	c, rs, err := websocket.Dial(ctx, destination, nil)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		err = c.Close(websocket.StatusNormalClosure, "closing connection")
```

Comment:
we should followup where we hold the connection open until an inactivity period.

Reply:
Added a follow up TODO